api_examples/get_vince.py

```diff
@@ -51,11 +67,12 @@ def create_response(key,turl):
         result[key] = json.loads(str(r.text))
 
 def vince_to_cvrf(vince):
+    case_id = vince["get_case"]["vuid"]
     cvrf =  { "document": {
         "acknowledgments": [
             {
                 "urls": [
-                    "https://kb.cert.org/vuls/id/"+vince["get_case"]["vuid"]
+                    "https://kb.cert.org/vuls/id/"+case_id
```

Comment:
This should include at least the anchor to acknowledgments section (as extracting the single values is not supported yet). The anchor must be supported by the website as well.
```suggestion
                    "https://kb.cert.org/vuls/id/"+case_id+"#Acknowledgements"
```